generator/resources/ec2_windows_test_matrix.json

```diff
@@ -1,8 +1,17 @@
 [
   {
     "os": "win-2019",
-    "instanceType":"g4dn.xlarge",
-    "ami": "Windows_Server-2019-English-Deep-Learning*",
-    "family": "windows"
+    "instanceType":"t3a.medium",
+    "ami": "cloudwatch-agent-integration-test-win-2019*"
```

Comment:
Adding win 2022 would not be possible here since I have the following error 
```
2022-03-11T16:15:28.946-0500 [ERROR] scp stderr: "At line:1 char:7\r\n+ "scp" -vt /tmp\r\n+ ~~~\r\nUnexpected token '-vt' in expression or statement.\r\nAt line:1 char:11\r\n+ "scp" -vt /tmp\r\n+ ~~~~\r\nUnexpected token '/tmp' in expression or statement.\r\n + CategoryInfo : ParserError: (:) [], ParentContainsErrorRecordEx \r\n ception\r\n + FullyQualifiedErrorId : UnexpectedToken\r\n \r\n"
```
[related discussion](https://github.com/hashicorp/terraform/issues/30661)